src/ArrayManager.hpp

```diff
@@ -218,6 +218,13 @@ class ArrayManager
    */
   PointerRecord* deepCopyRecord(PointerRecord const* record);
 
+  /*!
+   * \brief Create a copy of the pointers map.
```

Comment:
pointers -> pointer.

You might want to put a comment here about the expected use case, e.g. to check on memory usage, find leaks, etc.

Reply:
Done.